Logger/Handler/AdyenWarning.php

```diff
@@ -35,7 +33,7 @@ class AdyenWarning extends AdyenBase
     /**
      * @var int
      */
-    protected $loggerType = AdyenLogger::ADYEN_WARNING;
+    protected $loggerType = 300;
```

Comment:
Can you re-use the constants here, like `AdyenLogger::WARNING` instead of hard-coding the int?